lib/network.c

```diff
@@ -385,6 +386,16 @@ void h2op_forward_data_hook (H2OP_MSGTYPE type, nodeid_t source,
     }
 }
 
+void h2op_pump_set_data_hook (H2OP_MSGTYPE type, nodeid_t source,
```

Comment:
We could move all custom hooks to a different file and by this make the code a bit more modular. Then this file could just contain the basic network stack and we do not lose track of all the functions in here.